packages/block-library/src/gallery/edit.js

```diff
@@ -383,6 +383,7 @@ class GalleryEdit extends Component {
 				onError={ this.onUploadError }
 				notices={ hasImages ? undefined : noticeUI }
 				onFocus={ this.props.onFocus }
+				autoOpenMediaUpload={ ! hasImages && isSelected }
```

Comment:
Is it okay that I am passing a prop here that will only be used for the mobile component? I was wondering if there should be some sort of `if` logic or it's fine since the web will just ignore a prop it doesn't care about? LMK 🙏 
Also another point, I haven't finished setting up my local flow of how to test this PR in Gutenberg web so that is something that's necessary before these changes are merged.

Reply:
@guarani I tested all the changes made here with the local repo using `npm run wp-env start` from within the project folder and adding the Image, Video, and Gallery block showed no side effects.